src/main/java/no/nav/familie/integrasjoner/client/rest/DokarkivRestClient.kt

```diff
@@ -38,6 +36,16 @@ class DokarkivRestClient(@Value("\${DOKARKIV_V1_URL}") private val dokarkivUrl:
         }
     }
 
+    fun oppdaterJournalpost(jp: OppdaterJournalpostRequest, journalpostId: String): OppdaterJournalpostResponse {
+        val uri = UriUtil.uri(dokarkivUrl, PATH_JOURNALPOST, journalpostId)
+        try {
+            return putForEntity(uri, jp)!!
+        } catch (e: RuntimeException) {
+            secureLogger.error("Feil ved oppdatering av journalpost for bruker ${jp.bruker} ")
```

Comment:
Her bør du bruke OppslagsException. Har eget felt for sensitiveInfo som blir logget til securelogs